src/nexusrpc/_common.py

```diff
@@ -48,31 +48,62 @@ def __init__(
         message: str,
         *,
         type: HandlerErrorType,
-        retryable: Optional[bool] = None,
+        retry_behavior: Optional[HandlerErrorRetryBehavior] = None,
```

Comment:
The SDKs that are using an enumerate for this value have an "unspecified" as the unset/default instead of making this optional